engine/src/main/java/org/destinationsol/game/projectile/ProjectileConfigs.java

```diff
@@ -52,7 +54,13 @@ public ProjectileConfigs(OggSoundManager soundManager, EffectTypes effectTypes,
                 JSONObject node = rootNode.getJSONObject(s);
                 String name = s;
                 String texName = node.getString("tex");
-                TextureAtlas.AtlasRegion tex = Assets.getAtlasRegion(texName + "Projectile");
+                String projectileTexName = texName + "Projectile";
+                TextureAtlas.AtlasRegion tex = Assets.getAtlasRegion(projectileTexName);
+                Animation<TextureAtlas.AtlasRegion> animation = Assets.getAnimation(projectileTexName);
+                if (animation == null) {
+                    animation = new Animation<TextureAtlas.AtlasRegion>(0, tex);
+                }
```

Comment:
I'm seeing this block used a number or times. Could we create a helper function along the lines of `Assets.getAnimationOrDefault(texture)` to avoid the code duplication?

Reply:
`Assets.getAnimation` will now return an empty animation if no file exists for it. This animation is equivalent to a static frame consisting solely of the sprite texture provided to the method. The methods in the `Assets` class would normally fail if the asset does not exist, however with the animations most sprites will not have one, so it makes more sense to me to handle this as a fallback case for now.

If you want this in a helper function then I can change it. I personally think that not adding a new method would be better for code de-duplication though.